### DISCUSSION TITLE
Is iNat API allowing adding attributes?

### DISCUSSION THREAD

**Discussion by laser13** [rcloran/lr-inaturalist-publish#11] [Ideas]
<img width="509" alt="image" src="https://github.com/rcloran/lr-inaturalist-publish/assets/1689628/24ad7b4a-f60f-43dc-9772-154808638a94">

**Comment by rcloran** [comment]
Yes, it does. The API documentation is at https://api.inaturalist.org/v1/docs/#!/Annotations/post_annotations .

**Reply by laser13** [reply]
Maybe possible to add annotations section to export options, where possible to choose default values (or nothing)?
It would be convenient because 90% of my observation it is alive organisms.
<img width="530" alt="image" src="https://github.com/rcloran/lr-inaturalist-publish/assets/1689628/bbb897d6-90b1-4db5-b39b-8b029a347b9f">

**Reply by laser13** [reply]
The question, of course, is how to exclude the remaining 10% :)
